lists2safebrowsing.py

```diff
@@ -663,10 +663,17 @@ def start_versioning(config, chunknum, shavar_prod_lists_branches):
 
 
 def main():
+    execution_environment = os.getenv("EXECUTION_ENVIRONMENT", "JENKINS")
+    environment = os.getenv("ENVIRONMENT", "stage")
+
     config = configparser.ConfigParser()
     filename = config.read(["shavar_list_creation.ini"])
+
+    if execution_environment == "GKE":
+        filename = config.read([f"{environment}.ini"])
```

Comment:
Could we avoid reading the .ini file twice? If the `ENVIRONMENT` is set and the environment is GKE, we don't actually need to read from the `shavar_list_creation.ini` file.

Reply:
Made it into an if else statement!